news/2020-06-26-mappers-report-May-June.md

```diff
@@ -0,0 +1,193 @@
+
+### 2020 osu!catch Mapping World Cup
+
+Fruit catcher, now's your chance! The 2020 edition of the [osu!catch Mapping World Cup](https://osu.ppy.sh/community/forums/topics/1089125) has arrived! If you are skilled with placing fruits in the editor get your 2 to 5 best friends by their hand and sign up! Can the Philippines keep their title as osu!catch Mapping World Champion or will an other country get the 1st place this time?
+
+We will surely keep you up to date with it! 
+
+### lu^3 Mapping Cup
+
+Three time's the charm! The [lululu Mapping Cup](https://osu.ppy.sh/community/forums/topics/1086878) with it's 3rd edition has returned! This is a mapping contest for the osu! game mode only, featuring a specific song theme with every contest! This year's contest is using the "Swing Beat" theme.
+This contests has also something special. It can have two winners! One winner will be selected from the Taiwanese participants, the other winner will be selected by anyone. That's right, everyone is allowed to participate in the contest! 
+
+So what are you waiting for, grab the pre-timed .osz and join in! Supporter and maybe even a profile badge is waiting for you!
+
+Are there any other mapping contests going on? Or being announced soon? Please let us know! We would love to give it more attention to the public. No matter if it's mode or language specific. Head over to the [forum post](https://osu.ppy.sh/community/forums/topics/1027590) where you can find the submission form for it. 
+
+We hope to see a lot of exciting mapping contests in the future!
+
+## <a id="around-community"></a>Around the Community
+
+### New Review System in Beatmap Discussion
+
+Modders, listen. We have great news for you! As some may already noticed, modding got an update! The review system which got announced a while ago got finally added to the Beatmap Discussion! What it is you ask?
+
+Some may remember the old modding system which worked through the forum. People posted mods in mappers beatmap forum threads. The difference to the current modding system is that modder could bundle everything in one, huge post if they liked. A lot of modders liked it and still would like to have a similar modding possibility.
+
+And now they finally have! You can now bundle your modding in one, big post! Just like you could in the old beatmap threads. To access it, go into the beatmap discussion of any beatmap you like to mod and then go to the "Review" tab. That's all! Now you can mod like in the old days. Just with all the features which the current modding system offers you!
+
+You think it could be better and something is still missing? Or have you found any bug in it? The devs would love to hear that! Come join the [osu!dev Discord server](https://discord.gg/ppy) and join the discussion about the review system in the #modding channel! 
```

Comment:
```suggestion
Some may remember the old modding system which worked through the forum. People posted mods in mappers' beatmap forum threads. The difference to the current modding system is that modder could bundle everything in one, huge post if they liked. A lot of modders liked it and still would like to have a similar format when modding.

Now they finally do! You can now bundle your mods in one, big post! To access it, go into the beatmap discussion of any beatmap you like to mod and then go to the "Review" tab. That's all! Now you can mod in a format that lets you control what order the mod is read in, which is very useful for reoccurring or bigger issues in maps. Be sure to check out all the features which the current modding system offers to you!

Do you think it could be better and something is still missing? Or did you find a bug in it? The developrs would love to hear your feedback! Come join the [osu!dev Discord server](https://discord.gg/ppy) and join the discussion about the review system in the #modding channel! 
```